src/styles/_fonts.scss

```diff
@@ -0,0 +1,47 @@
+@font-face {
```

Comment:
FYI: for this we could actually potentially just load it from googles api, but I thought it might be better to have it directly downloaded and submitted with the app

Reply:
Not a fan though of assets being under src ( and not in the public folder )